src/openrct2/rct12/RCT12.cpp

```diff
@@ -925,6 +925,33 @@ uint8_t ConvertToTD46Flags(const TrackDesignTrackElement& source)
     return trackFlags;
 }
 
+void ImportMazeElement(TrackDesign& td, const TD46MazeElement& td46MazeElement)
+{
+    if (td46MazeElement.IsEntrance())
+    {
+        TrackDesignEntranceElement entrance{};
+        entrance.Location = TileCoordsXYZD(td46MazeElement.x, td46MazeElement.y, 0, td46MazeElement.Direction);
+        entrance.IsExit = false;
+        td.entrance_elements.push_back(entrance);
+    }
+    else if (td46MazeElement.IsExit())
+    {
+        TrackDesignEntranceElement exit{};
+        exit.Location = TileCoordsXYZD(td46MazeElement.x, td46MazeElement.y, 0, td46MazeElement.Direction);
+        exit.IsExit = true;
+        td.entrance_elements.push_back(exit);
+    }
```

Comment:
```suggestion
    if (td46MazeElement.IsEntrance() || td46MazeElement.IsExit())
    {
        TrackDesignEntranceElement element{};
        element.Location = TileCoordsXYZD(td46MazeElement.x, td46MazeElement.y, 0, td46MazeElement.Direction);
        element.IsExit = td46MazeElement.IsExit();
        td.entrance_elements.push_back(element);
    }
```
Maybe it can be simplified like this

Reply:
Done.